python/lsst/meas/algorithms/accumulator_mean_stack.py

```diff
@@ -106,25 +106,37 @@ def add_masked_image(self, masked_image, weight=1.0):
         good_pixels = np.where(((masked_image.mask.array & self.bit_mask_value) == 0)
                                & np.isfinite(masked_image.mask.array))
 
-        self.sum_weight[good_pixels] += weight
-        self.sum_wdata[good_pixels] += weight*masked_image.image.array[good_pixels]
+        weight_array = False
```

Comment:
I know this is 100% internal, but i'd suggest calling it `is_weight_array` to denote that its meant to be boolean.